sky/server/metrics.py

```diff
@@ -24,8 +24,10 @@
 metrics_app = fastapi.FastAPI()
 
 
+# Serve /metrics in dedicated thread to avoid blocking the event loop
+# of metrics server.
 @metrics_app.get('/metrics')
-async def metrics() -> fastapi.Response:
+def metrics() -> fastapi.Response:
```

Comment:
Curious, would this exhaust the threads in a uvicorn worker, considering people may have a lot of clusters.